test/Coverity/ConfigFiles/FreeRTOSConfig.h

```diff
@@ -81,7 +81,7 @@ void vConfigureTimerForRunTimeStats( void ); /* Prototype of function that initi
 #define portGET_RUN_TIME_COUNTER_VALUE()            ulGetRunTimeCounterValue()
 
 /* Co-routine related configuration options. */
-#define configUSE_CO_ROUTINES                     1
+#define configUSE_CO_ROUTINES                     0
```

Comment:
Can we remove these two configurations directly since the co-routines functionalities are removed in FreeRTOS-Kernel?

Reply:
yes, updated